README.md

```diff
@@ -1,75 +1,75 @@
 # NewPkgEval - Evaluate julia packages
 
 # Basic usage
-In order to run PkgEval against a julia do the following:
+In order to run PkgEval against a Julia Package do the following:
 
 
 1. Obtain NewPkgEval and install dependencies
-
-```
-git clone https://github.com/JuliaComputing/NewPkgEval.jl.git
-cd NewPkgEval.jl
-julia --project 'import Pkg; Pkg.instantiate()'
-```
+  
```

Comment:
These two spaces should probably be removed.

Reply:
That was the way it was formatted originally. I will defer to repo maintainers as to whether or not they want to keep that.

Reply:
I must be missing something. I'm referring to the whitespace (two space characters) in the blank line, which doesn't seem to be present in the current text. Can you please re-check?

Reply:
ah, I thought you were referencing the 2 newlines. I removed those spaces. Thanks!